lib/Parser/MimeDir.php

```diff
@@ -372,6 +372,13 @@ protected function readProperty($line)
                 $value = $this->unescapeParam($value);
 
                 if (is_null($lastParam)) {
+                    if ($this->options & self::OPTION_IGNORE_INVALID_LINES) {
+                        // When the property can't be matched and the configuration
+                        // option is set to ignore invalid lines, we ignore this line
+                        // This can happen when servers provide faulty data as iCloud
+                        //  frequently does with X-APPLE-SRTUCTURED-LOCATION
```

Comment:
```suggestion
                        //  frequently does with X-APPLE-STRUCTURED-LOCATION
```

I am not sure if the typo here is intentional - does iCloud have/had a bug where it uses `X-APPLE-SRTUCTURED-LOCATION` (with the typo?

Reply:
@heiglandreas is this just a typo?